snapshot-labs/snapshot
# Safe off-chain voting

Depend on:
- https://github.com/snapshot-labs/snapshot.js/pull/880
- https://github.com/snapshot-labs/lock/pull/99

## How to test:
- Add preview URL as a safe app
- Try voting on a space, that has the same network as the safe network
- It should by default open offchain voting (make sure to keep the voting modal open until everyone signs the message)
- Go to safe settings -> safe apps -> enable `Always use on-chain signatures`
- Now if you try to vote, it should open a on-chain voting (previous flow)

## TODO
- [x] Use production lock and snapshot.js packages
